.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+**Tasks**
+- [ ] Documentation (docs and inline)
+- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
+- [ ] Changelog
```

Comment:
should we also add naming convention?

Reply:
@philli-m sorry, not sure what you mean - naming convention for what?

Reply:
@goapunk naming convention of the pull request, with story no or ref and short description, helps finding them again